core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -780,6 +785,11 @@ public String requestContentPreview() {
     @Override
     public void requestContentPreview(@Nullable String requestContentPreview) {
         if (isAvailable(RequestLogProperty.REQUEST_CONTENT_PREVIEW)) {
+            if (requestContentPreview != null) {
+                logger.warn("You try to set the request content preview twice: {}. " +
+                            " Do you configure content previewing decorator more than once?",
+                            requestContentPreview);
```

Comment:
- Probably better logging only once?
- `{}.` -> `{}` ? (The resulting log message would be: `SomePreview.` looking like `.` is part of the preview)
- You try -> You tried
- Do you configure .. -> Did you apply .. ?

Reply:
Fixed. Thanks!